portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -22,13 +30,20 @@
 import java.util.ArrayList;
 import java.util.List;
 import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
```

Comment:
Comments and facts are two different data sets. In this case, it's prefer to to put them into same API.

We can create:
@WebServlet("/comment")
public class CommentServlet extends HttpServlet
@WebServlet("/fact")
public class FactServlet extends HttpServlet

Reply:
Few comments about that.. 
1. The walkthrough kind of made it seem that they wanted it all on the same server, so that's why I implemented it that way. Maybe I interpreted it wrong though.
2. Does @WebServlet("/comment") go in a different java file or does the @ tag differ between the different servers
3. In order for me to make the change between all the @ tags for the different servers, I would have to change the implementation of my doGet and doPost which would take a while... could I safe that change for a different pull request just so I can finish the last part of this week's requirements?

Reply:
> The walkthrough kind of made it seem that they wanted it all on the same server, so that's why I implemented it that way. Maybe I interpreted it wrong though.

I haven't read the walk-through in every  words... It seems to me fun fact generator is a intermediate step. Once we have comment in place, it can be removed.

> Does @WebServlet("/comment") go in a different java file or does the @ tag differ between the different servers.

Yes. I think java allow 1 public class in 1 file. 
This is an good example:
https://github.com/googleinterns/step/tree/master/walkthroughs/week-3-server/examples/todo-list/src/main/java/com/google/sps/servlets

> could I safe that change for a different pull request just so I can finish the last part of this week's requirements?
Yeap, you can have a separate change request for it, most of the changes in this pull request would be overwrite by new pull requests. I'll skip review on this one and review the next one in this case.
Let me know how you want to proceed.

Reply:
Okay thanks... what I'll do is save that differing server change for a different pull request. 
I'll resolve the other comment and make a new commit to this and then I'll get to trucking on step 5.

Reply:
Sounds good.